src/InputWithLabel.jsx

```diff
@@ -0,0 +1,20 @@
+import { useEffect, useRef } from "react";
+
+export default function InputWithLabel({ children, value, handleChange }) {
+  const inputRef = useRef();
+
+  useEffect(() => inputRef.current.focus());
```

Comment:
```
useEffect(() => {
  inputRef.current.focus();
}, []);
```

This prevents the focus operation from running on every re-render and only runs it once when the component mounts.

Reply:
The assignment actually asks for the useEffect to not have any dependencies so that it can be called on every render.